examples/customresources/demos/complexmodule/module.json

```diff
@@ -2,7 +2,7 @@
 	"modules": [
 		{
 			"name": "AcmeModule",
-			"executable_path": "./run.sh",
+			"executable_path": "./complexmodule",
```

Comment:
Why are we avoiding the `run.sh` now?

Reply:
The "trick" with run.sh was that it builds the module as part of module-startup, though I know we've mostly worked around that. Still, if we're not using it, run.sh should be deleted to avoid confusion, yeah? I also think all examples should do it the same way. (Either all direct, or all with run.sh) Most other areas of code I wouldn't nitpick about consistency, but for examples, I expect outside user to be in here learning somewhat regularly, so it's important we keep things clean, IMHO.

Reply:
not super important so I think I'll just make it all use `run.sh`